test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -239,52 +239,58 @@ std::string RandomCodeBase::rndByteSequence(int _length, SizeStrictness _sizeTyp
 	return hash;
 }
 
+bool isOpcodeDefined(uint8_t _opcode)
+{
+	eth::Instruction inst = (eth::Instruction) _opcode;
+	eth::InstructionInfo info = eth::instructionInfo(inst);
+	if (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
+		|| std::find(invalidOpcodes.begin(), invalidOpcodes.end(), inst) != invalidOpcodes.end())
+		return false;
+	return true;
+}
+
+uint8_t makeOpcodeDefined(uint8_t _opcode)
+{
+	while (!isOpcodeDefined(_opcode))
+		_opcode++; //Byte code is yet not implemented. Try next one.
+	return _opcode;
+}
+
 //generate smart random code
 std::string RandomCodeBase::generate(int _maxOpNumber, RandomCodeOptions const& _options)
 {
 	std::string code;
-
 	if (test::RandomCode::get().randomPercent() < _options.emptyCodeProbability)
 		return code;
 
-	//random opCode amount
+	//generate [0 ... _maxOpNumber] opcodes.
 	int size = (int)(test::RandomCode::get().randomPercent() * _maxOpNumber / 100);
 	assert(size <= _maxOpNumber);
 
 	for (auto i = 0; i < size; i++)
 	{
 		uint8_t opcode = _options.getWeightedRandomOpcode();
-		eth::Instruction inst = (eth::Instruction) opcode;
-		eth::InstructionInfo info = eth::instructionInfo(inst);
-
-		while (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
-			|| std::find(invalidOpcodes.begin(), invalidOpcodes.end(), inst) != invalidOpcodes.end())
+		if (!isOpcodeDefined(opcode) && _options.useUndefinedOpCodes)
 		{
-			if (_options.useUndefinedOpCodes)
+			code += toCompactHex(opcode, 1);
+			continue;
```

Comment:
Here, the execution enters an infinite loop.

I think the execution should go out of the while loop.

Reply:
this continue goes to the for loop. it wont be infinite

Reply:
I think you are mistaken what `continue` does in nested loops.

Reply:
I have removed the nested loop. there is makeOpcodeDefined func now